Remove Matter from sdk-nrf (#30253)
nrfconnect/sdk-nrf: doc/nrf/releases_and_maturity/known_issues.rst

Matter
======

> The issues in this section are related to the ``ug_matter`` protocol.

Reviewer: ```suggestion
The issues in this section are related to the :ref:`ug_matter` protocol.
Starting with the |NCS| v3.5.0 release, known issues are moved to the `Matter Add-on`_ repository.
```

Reviewer: Seems like this is not fixed.

.. rst-class:: v3-1-1 v3-0-2 v3-0-1 v3-0-0

> KRKNWK-20948: The Matter west ``Matter GS Tools Matter West Commands Zap Tool Generate`` might not work properly on Windows

Reviewer: ```suggestion
KRKNWK-20948: The Matter west ``zap-generate`` command might not work properly on Windows
```

> KRKNWK-20948: The Matter west ``ug_matter_gs_tools_matter_west_commands_zap_tool_generate`` might not work properly on Windows

Reviewer: ```suggestion
KRKNWK-20948: The Matter west ``zap-generate`` command might not work properly on Windows
```
Could covert to plain text instead of double backticks.

.. rst-class:: v3-1-0 v3-0-2 v3-0-1 v3-0-0

KRKNWK-20815: The NordicDevKit cluster does not work with the 0.1.0 version of the `Matter Cluster Editor app`_

>   The ``NordicDevKit`` cluster from the ``matter_manufacturer_specific_sample`` is not supported in the version 0.1.0 of the `Matter Cluster Editor app`_.

Reviewer: ```suggestion
  The ``NordicDevKit`` cluster from the Matter: Manufacturer-specific sample is not supported in the version 0.1.0 of the `Matter Cluster Editor app`_.
```
The same applies to all the samples and user guide ref links in the Known issues page.

  The DFU fails for nRF5340 DK, if the application enables the :kconfig:option:`CONFIG_RAM_POWER_DOWN_LIBRARY` Kconfig option.
  This option is enabled by default for the ``release`` configuration of the following samples:

    * Matter Lock sample:
    * Matter Light Switch sample:
    * Matter Smoke Co Alarm sample:

>     * Matter Window Covering sample:

Reviewer: ```suggestion
    * Matter Lock sample
    * Matter Light Switch sample
    * Matter Smoke Co Alarm sample
    * Matter Window Covering sample
```

.. rst-class:: v2-1-4 v2-1-3 v2-1-2 v2-1-1 v2-1-0

KRKNWK-14473: Unreliable communication with the window covering sample

>   The ``matter_window_covering_sample`` might rarely become unresponsive for a couple of seconds after commissioning to the Matter network.

Reviewer: ```suggestion
  The Matter: Window covering sample might rarely become unresponsive for a couple of seconds after commissioning to the Matter network.
```
